README.md

```diff
@@ -1,565 +1,40 @@
-- **IPLD Data Model**: a flexible, JSON-based data model for representing merkle-dags.
-- **IPLD Serialized Formats**: a set of formats in which IPLD objects can be represented, for example JSON, CBOR, CSON, YAML, Protobuf, XML, RDF, etc.
-- **IPLD Canonical Format**: a deterministic description on a serialized format that ensures the same _logical_ object is always serialized to _the exact same sequence of bits_. This is critical for merkle-linking, and all cryptographic applications.
+**Specs are not finished yet. We use the following tag system to identify their state:**
 
-In short: JSON documents with named merkle-links that can be traversed.
+- ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) - this spec is a work-in-progress, it is likely not even complete.
+- ![](https://img.shields.io/badge/status-draft-yellow.svg?style=flat-square) - this spec is a rough draft and will likely change substantially.
+- ![](https://img.shields.io/badge/status-reliable-green.svg?style=flat-square) - this spec is believed to be close to final. Minor changes only.
+- ![](https://img.shields.io/badge/status-stable-brightgreen.svg?style=flat-square) - this spec is likely to improve, but not change fundamentally.
+- ![](https://img.shields.io/badge/status-permanent-blue.svg?style=flat-square) - this spec will not change.
 
-## Intro
+Nothing in this spec repository is `permanent` yet. The most important pieces of IPLD are now `reliable` or `stable`. Many subsystems remain as `draft`.
 
-### What is a _merkle-link_?
+Note that, as in many IPFS repositories, most of the work is happening in [the issues](https://github.com/ipld/specs/issues/) or in [active pull requests](https://github.com/ipld/specs/pulls/). Go take a look!
 
-A _merkle-link_ is a link between two objects which is content-addressed with the _cryptographic hash_ of the target object, and embedded in the source object. Content addressing with merkle-links allows:
+## Specs
 
-- **Cryptographic Integrity Checking**: resolving a link's value can be tested by hashing. In turn, this allows wide, secure, trustless exchanges of data (e.g. git or bittorrent), as others cannot give you any data that does not hash to the link's value.
-- **Immutable Datastructures**: data structures with merkle links cannot mutate, which is a nice property for distributed systems. This is useful for versioning, for representing distributed mutable state (eg CRDTs), and for long term archival.
+The specs contained in this repository are:
 
-A _merkle-link_ is represented in the IPLD object model by a map containing a single key `/` mapped to a "link value". For example:
+**IPLD:**
+- [IPLD](/ipld) - spec about the data model, pointers and link formats
+- IPLD Selectors (not started) - spec about simple language to select multiple unknown nodes in a graph
+- IPLD Transformations (not started) - spec about the language to trasform an IPLD graph into another
```

Comment:
in the IPFS repo, badges are added to the actual spec page, do we want them here too?